config/params_template.env

```diff
@@ -40,15 +40,13 @@ export src_subdiv_toggle="True" # Toggle to run composite roughness src routine
 export vrough_suffix="" # text to append to output log and hydrotable file names (use for testing/debugging)
 export vmann_input_file="/data/inputs/rating_curve/variable_roughness/mannings_global_06_12.csv" # input file location with nwm feature_id and channel roughness and overbank roughness attributes
 
-#### apply SRC adjustments using USGS rating curve database (calibration db) ####
+#### apply SRC adjustments using USGS rating curve database (calibration points db) ####
```

Comment:
This is block is for the USGS rating curve calibration (not the point data base)

Reply:
Good Catch! Thanks.